base/workflow/R/run.write.configs.R

```diff
@@ -211,13 +228,26 @@ run.write.configs <- function(settings, ensemble.size, input_design, write = TRU
   PEcAn.logger::logger.info("###### Finished writing model run config files #####")
   PEcAn.logger::logger.info("config files samples in ", file.path(settings$outdir, "run"))
 
-  ### Save output from SA/Ensemble runs
-  # A lot of this is duplicate with the ensemble/sa specific output above, but kept for backwards compatibility.
-  save(ensemble.samples, trait.samples, sa.samples, runs.samples, pft.names, trait.names,
-    file = file.path(settings$outdir, "samples.Rdata")
-  )
-  PEcAn.logger::logger.info("parameter values for runs in ", file.path(settings$outdir, "samples.RData"))
+  # write runs manifest
+
+  manifest.file <- file.path(settings$outdir, "runs_manifest.csv")
+
+  if (nrow(run_manifest_df) > 0) {
+    if (overwrite && file.exists(manifest.file)) {
+      unlink(manifest.file)
+    }
+
+    utils::write.table(run_manifest_df,
+                       file = manifest.file,
+                       sep = ",",
+                       row.names = FALSE,
+                       col.names = !file.exists(manifest.file),
+                       append = file.exists(manifest.file))
```

Comment:
write.table removes existing files unless append=TRUE, so this unlink isn't needed
```suggestion
    utils::write.table(run_manifest_df,
                       file = manifest.file,
                       sep = ",",
                       row.names = FALSE,
                       col.names = !file.exists(manifest.file),
                       append = !overwrite)
```

Reply:
exactly, its defensive that adds complexity without benefit